docs/src/use/troubleshooting/index.md

```diff
@@ -0,0 +1,16 @@
+---
+title: Troubleshooting
+eleventyNavigation:
+    key: core concepts
+    title: Troubleshooting
+    parent: use eslint
+    order: 8
+---
+
+This page serves as a reference for common issues working with ESLint.
```

Comment:
I couldn't think of what else to say here but feel weird about it being so terse. 😬

Reply:
Oh I like that!